### DISCUSSION TITLE
Locker Password Manager Roadmap 2024

### DISCUSSION THREAD

**Discussion by issimonsi** [lockerpm/.github#5] [Announcements]
This Roadmap is created based on the user's vote here: https://forum.locker.io/d/64-vote-for-the-feature-you-want-implemented-prepare-for-roadmap-2024/19

Roadmap link: https://cystack.notion.site/Locker-Password-Manager-Roadmap-2024-d86f8ff0fed24b9388c8533c8101aa4f

**Comment by snhv** [comment]
Hi, I'm not familiar with Notion. Can we vote for features there?

**Reply by issimonsi** [reply]
Hi,

We use Notion as the official channel for sharing information, but users can't edit or comment there. The deadline for voting on the 2024 roadmap has passed, but we still get new feature requests. You can now vote for features in GitHub polls.

**Comment by snhv** [comment]
Hi @issimonsi, are you still using that link (Roadmap for 2024) or is there a new roadmap for 2025?

**Reply by daidynguyen** [reply]
Hi @snhv,

You can find our updated 2025 roadmap here: https://cystack.notion.site/Locker-Password-Manager-Roadmap-2025-6e19100d7379492da0f2eb311ceb4350.
It’s also available publicly on our website for your convenience. Thanks!
